util/chplenv/chpl_bin_subdir.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import optparse
+import os
+from sys import stderr, stdout
```

Comment:
There is only one use of stdout here, so I'd prefer to just stick with fully qualified names